coldfront/plugins/qumulo/views/allocation_view.py

```diff
@@ -29,8 +29,9 @@
 
 class AllocationView(LoginRequiredMixin, FormView):
     form_class = AllocationForm
-    template_name = "allocation.html"
     new_allocation = None
+    template_name = "allocation.html"
```

Comment:
I'd like to see this returned to the 2nd line of the class.  Reason: It's part of the standard Django form template setup and I think it makes sense to keep those required template/config variables inherit to the Djangoness of it all at the top

Reply:
Unresolving because the was not changed or responded to